zio-http/src/main/scala/zhttp/service/server/ServerChannelInitializer.scala

```diff
@@ -70,6 +75,12 @@ final case class ServerChannelInitializer[R](
     // Flushing content is done in batches. Can potentially improve performance.
     if (cfg.consolidateFlush) pipeline.addLast(HTTP_SERVER_FLUSH_CONSOLIDATION, new FlushConsolidationHandler)
 
+    if (Log.isEnabled) {
```

Comment:
we should be using `log` not `Log` over here.

Reply:
Done